styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -157,6 +162,38 @@ public String start(WorkflowInstance workflowInstance, RunSpec runSpec) throws I
     }
   }
 
+  @Override
```

Comment:
Good point.

Perhaps we should just always remove all SA keys & secrets (perhaps excluding those used by currently booting pods). Then we wouldn't keep keys for "idle" / disabled workflows and just create them as needed. We'd also achieve key "rotation" for free...

🤔